spikes/cockpit-ux-spike/styles.css

```
@@ -0,0 +1,495 @@
+.dock-tabs { display: flex; gap: 2px; }
+.dock-tab {
+  font-size: 11px; font-weight: 600; letter-spacing: .03em; text-transform: uppercase;
+  color: var(--muted); cursor: pointer; padding: 0 4px;
+}
+.dock-tab.on { color: var(--ink); }
+.dock-tab.on::after { content: ""; display: block; height: 2px; background: var(--accent); margin-top: 6px; border-radius: 2px; }
+
+.ws-wrap { position: relative; flex: 1; min-height: 0; }
+.ws-pre, .ws-ta {
+  margin: 0; padding: 12px 14px; font-family: var(--code-font, monospace);
+  font-size: 12.5px; line-height: 1.6; white-space: pre-wrap; word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)